src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -272,7 +275,8 @@ private void WriteValue(Value value, bool isFirst = true, bool valueInList = fal
 
         private void WriteValue(string value)
         {
-            _writer.Write($"\"{value}\"");
+            _writer.Flush();
+            s_jsonStringSerializer.WriteObject(_writer.BaseStream, value);
```

Comment:
@heejaechang The DataContractJsonSerializer.WriteObject overload that takes a writer is not supported on portable (see [here](https://msdn.microsoft.com/en-us/library/system.runtime.serialization.json.datacontractjsonserializer.writeobject(v=vs.110).aspx)).

For now, I am just going to make this an instance field to address the thread safety.